src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -882,4 +863,65 @@ public double getSaturation()
     {
         return this.saturation;
     }
+
+    /**
+     * Getter for the saturation.
+     *
+     * @param extraSaturation the extra saturation
+     */
+    public void increaseSaturation(final double extraSaturation)
+    {
+        this.saturation = Math.min(MAX_SATURATION, this.saturation + Math.abs(extraSaturation));
+    }
+
+    /**
+     * Getter for the saturation.
+     *
+     * @param extraSaturation the saturation to remove.
+     */
+    public void decreaseSaturation(final double extraSaturation)
+    {
+        this.saturation = Math.max(MIN_SATURATION, this.saturation - Math.abs(extraSaturation));
+    }
+
+    /**
+     * Resets the experience and the experience level of the citizen.
+     */
+    public void resetExperienceAndLevel()
+    {
+        this.level = 0;
+        this.experience = 0;
+    }
+
+    public InventoryCitizen getInventory()
+    {
+        return inventory;
+    }
+
+    public <Request extends IRequestable> IToken createRequest(@NotNull Request requested)
+    {
+        return getWorkBuilding().createRequest(this, requested);
+    }
+
+    public <Request extends IRequestable> IToken createRequestAsync(@NotNull Request requested)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00119)